deploy/clowdapp.yaml

```diff
@@ -258,9 +252,6 @@ parameters:
   - description: Determines Clowder deployment
     name: CLOWDER_ENABLED
     value: "true"
-  - description: AWS CloudWatch logging integration
-    name: CLOUDWATCH_ENABLED
-    value: "false"
```

Comment:
I noticed this, this is now unused. Cloudwatch is configured directly from clowder.